pkg/webhooks/server.go

```diff
@@ -153,3 +153,31 @@ func filter(configuration config.Configuration, inner handlers.AdmissionHandler)
 func admission(logger logr.Logger, monitor *webhookconfig.Monitor, inner handlers.AdmissionHandler) http.HandlerFunc {
 	return handlers.Monitor(monitor, handlers.Admission(logger, protect(inner)))
 }
+
+func registerWebhookHandlers(
+	logger logr.Logger,
+	mux *httprouter.Router,
+	basePath string,
+	monitor *webhookconfig.Monitor,
+	configuration config.Configuration,
+	handlerFunc func(logr.Logger, *admissionv1.AdmissionRequest, string, time.Time) *admissionv1.AdmissionResponse,
+) {
+	mux.HandlerFunc("POST", basePath, admission(logger, monitor, filter(
+		configuration,
+		func(logger logr.Logger, request *admissionv1.AdmissionRequest, startTime time.Time) *admissionv1.AdmissionResponse {
+			return handlerFunc(logger, request, "all", startTime)
+		})),
```

Comment:
To be backward compatible. Let's remove it in a 1.9 for example.

Reply:
So now we apply policies three times?

Reply:
Oh no, we are targeting the specific ones in webhook configs.
I can remove it if you prefer, I fear that it will cause issues while upgrading though.

Reply:
To avoid issues, we should only switch once all pods have been rolled out.

Reply:
Done